packages/electron-updater/src/util.ts

```diff
@@ -35,3 +36,15 @@ export function blockmapFiles(baseUrl: URL, oldVersion: string, newVersion: stri
   const oldBlockMapUrl = newUrlFromBase(`${baseUrl.pathname.replace(new RegExp(escapeRegExp(newVersion), "g"), oldVersion)}.blockmap`, baseUrl)
   return [oldBlockMapUrl, newBlockMapUrl]
 }
+
+export function execShellCommand(cmd: string): Promise<{ error: ExecException | null; stdout: string | Buffer; stderr: string | Buffer }> {
+  return new Promise((resolve, reject) => {
+    try {
+      exec(cmd, (error, stdout, stderr) => {
+        resolve({ error, stdout, stderr })
```

Comment:
`exec` can error without outputting a `stderr`. So, you probably want to check for `error` object in MacUpdater.ts, or simply throw which should reject the promise.
```suggestion
        if (error) throw error
        resolve({ stdout, stderr })
```